packages/expo-video/ios/VideoPlayer.swift

```diff
@@ -143,9 +143,7 @@ internal final class VideoPlayer: SharedRef<AVPlayer>, Hashable, VideoPlayerObse
       let videoSource = videoSource,
       let url = videoSource.uri
     else {
-      DispatchQueue.main.async { [pointer] in
-        pointer.replaceCurrentItem(with: nil)
-      }
+      pointer.replaceCurrentItem(with: nil)
```

Comment:
Please revert this change, it does not appear to be intended as part of your PR.